### DISCUSSION TITLE
Backend 'setuptools.build_meta:__legacy__' is not available.

### DISCUSSION THREAD

**Discussion by smartalecH** [jax-ml/jax#17598] [Q&A]
I'm working on building jaxlib from source. Everything compiles fine, but I get an error when trying to build the jaxlib wheel:

```
Backend 'setuptools.build_meta:__legacy__' is not available.
```

This is a pretty [common issue](https://github.com/pypa/setuptools/issues/1694) when using `pyproject-tools` and `build` and when one forgets to specify the `build-backend` in the `pyproject.toml` for "complicated systems".

I noticed that the primary jax module has a `pyproject.toml` with the `build-backend` explicitly defined. But jaxlib is using a `setup.py` file, and its not obvious to me if I can somehow explicitly specify the `build-backend` this way? Any ideas on how I can circumvent this? Thanks so much!

**Comment by hawkinsp** [comment]
Does `pip install setuptools` fix the problem?

**Reply by smartalecH** [reply]
Thanks @hawkinsp, for chiming in! Yep I have that already in my env, but I get the same issue. I opened a REPL session just to make sure all the required dependencies were properly loaded. 

I also tried manually adding the `pyproject.toml` file to the jaxlib build target dependencies in the bazel BUILD file, but that doesn't seem to help either.
